scripts/migration/migrate_twitter_handles.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from nose.tools import *
+from tests.base import OsfTestCase
+from tests.factories import UserFactory
+from website.app import init_app
+from website.models import User
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+def get_targets():
+    return User.find(Q('social.twitter', 'contains', '@'))
+
+def migrate(targets, dry_run=True):
+    # iterate over targets
+    # log things
+    users = targets
+    logging.info(len(users))
+    for user in users:
+        twitter = user.social['twitter'].replace("@", "")
+        logger.info('Setting `social.twitter` field of user {0} to {1}'.format(user._id, twitter))
+        if not dry_run:
+            user.social['twitter'] = twitter
+            user.save()
+    '''
+    if dry_run:
+        raise RuntimeError('Dry run, transaction rolled back.')
+    '''
+
+def main():
+    dry_run = False
+    if '--dry' in sys.argv:
+        dry_run = True
+    if not dry_run:
```

Comment:
Not a big deal, but you could change this to `else:` instead, since it's right there.